.github/workflows/build-test-deploy.yml

```diff
@@ -832,8 +830,18 @@ jobs:
       - run: |
           cachix watch-exec --jobs 16 --compression-level $CACHIX_COMPRESSION_LEVEL composable-community nix -- build .#devnet-container --no-update-lock-file --show-trace -L
           docker load --input ./result
-          docker tag composable-devnet-container:latest "${{ env.DOCKER_REGISTRY_NAME }}/composable-sandbox:latest"
-          docker push "${{ env.DOCKER_REGISTRY_NAME }}/composable-sandbox:latest"
+          # update soon https://github.com/actions/checkout/issues/760
+          git config --global --add safe.directory /__w/composable/composable
+          CONTAINER_NAME=composable-sandbox
+          SHA256=$(sha256sum ./result | cut --delimiter " " --fields 1)
+          COMMIT_SHA=$(git rev-parse HEAD)
+          DOCKER_IMAGE_HASH=$(docker images | tail --lines=1 | tr --squeeze-repeats  ' ' |  cut --delimiter " " --fields 3)
```

Comment:
there is `composable-devnet-container:latest` docker image when the docker image is built.
so, you just can use `composable-devnet-container:latest` instead of `DOCKER_IMAGE_HASH`.

Reply:
i am worry that can change nix container name. then this thing will fail.
while i rely on fact that system is clean, and has only one just built container. that also can fail. 

not sure what is better. may be need to change to name based approach.

Reply:
> i am worry that can change nix container name

why we can want to do it? we can just add a new tag  `composable-devnet-container:latest` that we need.

imho, `composable-devnet-container:latest` is read simler that `DOCKER_IMAGE_HASH=$(docker images | tail --lines=1 | tr --squeeze-repeats  ' ' |  cut --delimiter " " --fields 3)`